MekHQ/data/scenariotemplates/Recon.xml

```diff
@@ -104,8 +106,11 @@
                     <howMuch>1</howMuch>
                 </failureEffect>
             </failureEffects>
-            <additionalDetails/>
-            <description>Scan the following enemy unit(s) and force(s). Scans may be conducted by weapons fire, TAG, spotting or physical attacks. If the scan action requires a roll, it must be possible:</description>
+            <additionalDetails>
+                <additionalDetail>Scans may be conducted by weapons fire, TAG, spotting or physical attacks.</additionalDetail>
+                <additionalDetail>If the scan action requires a roll, it must be possible.</additionalDetail>
```

Comment:
The old text ended in a colon, was that unintentional?

Reply:
Probably not, let me double check.

Reply:
Ah, ok. The serializer alphabetized the property names. If you look at "description", it still has the colon at the end. It looks weird on the scenario resolution screen when the description is too long for a custom objective type, so I solved the problem by shortening the description. :D